ops/jenkins/global-pipeline-libraries/vars/awsSsm.groovy

```diff
@@ -18,11 +18,16 @@ String putParameter(Map args = [:]) {
     name = args.parameterName
     value = args.parameterValue
     type = args.parameterType ?: 'String'
+    tags = args.parameterTags ?: "Key=Source,Value=${JOB_NAME} Key=Environment,Value=mgmt Key=stack,Value=mgmt Key=Terraform,Value=False Key=application,Value=bfd Key=business,Value=oeda"
     overwrite = args.shouldOverwrite ? '--overwrite' : ''
     awsRegion = args.awsRegion ?: 'us-east-1'
     includeType = "--type ${type}"
 
+    rType = args.resourceType ?: 'Parameter'
+    rId = args.resourceId ?: name
+
     // TODO this is very naive and there are a crazy number of cases that this does not support. Beware.
-    output = sh(returnStdout: true, script: "aws ssm put-parameter --name ${name} --value '${value}' ${includeType} --region ${awsRegion} ${overwrite}").trim()
-    return output
+    parameterOutput = sh(returnStdout: true, script: "aws ssm put-parameter --name ${name} --value '${value}' ${includeType} --region ${awsRegion} ${overwrite}").trim()
+    tagOutput = sh(returnStdout: true, script: "aws ssm add-tags-to-resource --resource-type ${rType} --resource-id ${rId} --tags ${tags}").trim()
```

Comment:
I still think this should be a separate function like you had before - `addTagsToResource`, should keep that separation of functionality. then just call `addTagsToResource` here. this exposes the functionality which could be useful down the line, and keeps the library modular